apps/studio/components/interfaces/Auth/Users/UserOverview.tsx

```diff
@@ -0,0 +1,500 @@
+import { useUserResetPasswordMutation } from 'data/auth/user-reset-password-mutation'
+import { useUserSendMagicLinkMutation } from 'data/auth/user-send-magic-link-mutation'
+import { useUserSendOTPMutation } from 'data/auth/user-send-otp-mutation'
+import { User } from 'data/auth/users-query'
+import { useCheckPermissions } from 'hooks/misc/useCheckPermissions'
+import { BASE_PATH } from 'lib/constants'
+import { timeout } from 'lib/helpers'
+import { Admonition, Button, cn, Separator } from 'ui'
+import ConfirmationModal from 'ui-patterns/Dialogs/ConfirmationModal'
+import { PROVIDERS_SCHEMAS } from '../AuthProvidersFormValidation'
+import { PANEL_PADDING } from './UserPanel'
+import { getDisplayName, providerIconMap } from './Users.utils'
+
+const DATE_FORMAT = 'DD MMM, YYYY HH:mm'
+const CONTAINER_CLASS = cn(
+  'bg-surface-100 border-default text-foreground flex items-center justify-between',
+  'gap-x-4 border px-5 py-4 text-sm first:rounded-tr first:rounded-tl last:rounded-br last:rounded-bl'
+)
+
+interface UserOverviewProps {
+  user: User
+  onDeleteSuccess: () => void
+}
+
+export const UserOverview = ({ user, onDeleteSuccess }: UserOverviewProps) => {
+  const { ref: projectRef } = useParams()
+  const displayName = getDisplayName(user)
+  const hasDisplayName = displayName !== '-'
+  const isEmailAuth = user.email !== null
+  const isPhoneAuth = user.phone !== null
+  const isAnonUser = user.is_anonymous
+
+  const providers = (user.raw_app_meta_data?.providers ?? []).map((provider) => {
+    return {
+      name: provider,
+      icon:
+        provider === 'email'
+          ? `${BASE_PATH}/img/icons/email-icon2.svg`
+          : providerIconMap[provider]
+            ? `${BASE_PATH}/img/icons/${providerIconMap[provider]}.svg`
+            : undefined,
+    }
+  })
+
+  const canSendMagicLink = useCheckPermissions(PermissionAction.AUTH_EXECUTE, 'send_magic_link')
+  const canSendRecovery = useCheckPermissions(PermissionAction.AUTH_EXECUTE, 'send_recovery')
+  const canSendOtp = useCheckPermissions(PermissionAction.AUTH_EXECUTE, 'send_otp')
+  const canRemoveUser = useCheckPermissions(PermissionAction.TENANT_SQL_DELETE, 'auth.users')
+  const canRemoveMFAFactors = useCheckPermissions(
+    PermissionAction.TENANT_SQL_DELETE,
+    'auth.mfa_factors'
+  )
+
+  const [successAction, setSuccessAction] = useState<
+    'send_magic_link' | 'send_recovery' | 'send_otp'
+  >()
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false)
+  const [isDeleteFactorsModalOpen, setIsDeleteFactorsModalOpen] = useState(false)
+
+  const { data } = useAuthConfigQuery({ projectRef })
+  const mailerOtpExpiry = data?.MAILER_OTP_EXP ?? 0
+  const minutes = Math.floor(mailerOtpExpiry / 60)
+  const seconds = Math.floor(mailerOtpExpiry % 60)
+  const formattedExpiry = `${mailerOtpExpiry > 60 ? `${minutes} minutes ${seconds > 0 ? 'and' : ''} ` : ''}${seconds > 0 ? `${seconds} seconds` : ''}`
```

Comment:
Will minutes / seconds ever be `1`? If so we should handle the singular language here